src/screens/CreateWallet/screens/CreateWalletStep2/MnemonicSelected.tsx

```diff
@@ -12,16 +12,20 @@ interface MnemonicSelectedProps {
 
 export const MnemonicSelected = (props: MnemonicSelectedProps) => {
   const { onPress, word, index } = props;
+  const isValue = useMemo(() => {
+    return !!word;
+  }, [word]);
```

Comment:
const isValue = useMemo(() => {
    return !!word;
  }, [word]); -> 
  
  const isValue = useMemo(() => !!word, [word]);
  
  It can be one line function

Reply:
logic changed